ServiceCommon/Classes/TestCustomerEventConsumer.cs

```diff
@@ -0,0 +1,52 @@
+﻿using IO.Eventuate.Tram.Events.Subscriber;
+using IO.Eventuate.Tram.Messaging.Common;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ServiceCommon.Classes
+{
+    public class TestCustomerEventConsumer : IDomainEventHandler<CustomerCreatedEvent>, IDomainEventHandler<CustomerValidationFailedEvent>, IDomainEventHandler<CustomerCreditReservedEvent>, IDomainEventHandler<CustomerCreditReservationFailedEvent>
```

Comment:
There should be a single TestEventConsumer class that has a single BlockingCollection.
This would be simpler -- that's how the Java code works.

Reply:
Added single TestEventConsumer and single List<IDomainEvent>